### PR TITLE
Improved e2e log access

### PR DESCRIPTION
**Pull request by yourheropaul**
This uploads e2e logs as a .zip file, and includes the public URL to the file in the slack notification. The slack notification has also been fixed (it was missing a secret).

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -493,7 +501,7 @@ pipeline:
 
         _${DRONE_COMMIT_MESSAGE}_
 
-        {{#if build.tag}}`v{{build.tag}}` | {{/if}}Commit <${DRONE_COMMIT_LINK}|{{build.commit}}> | <{{build.link}}|Build #{{build.number}}>
+        {{#if build.tag}}`v{{build.tag}}` | {{/if}}Commit <${DRONE_COMMIT_LINK}|{{build.commit}}> | <{{build.link}}|Build #{{build.number}}> | <https://s3.amazonaws.com/kcoin-e2e/logs/${DRONE_BUILD_NUMBER}/e2e-build-${DRONE_BUILD_NUMBER}.zip|e2e logs>
```
Comment (acroca) [inline .drone.yml]: You have to change the acl of the upload to `public`

Comment (yourheropaul) [reply .drone.yml]: The whole bucket is public

Comment (yourheropaul) [reply .drone.yml]: Oh wait, maybe not.

Comment (acroca) [reply .drone.yml]: The current log files show `Access Denied`

Comment (yourheropaul) [reply .drone.yml]: There.

**Pull request merged**